ppocr/metrics/rec_metric.py

```diff
@@ -41,41 +41,56 @@ def __call__(self, pred_label, *args, **kwargs):
         correct_num = 0
         all_num = 0
         norm_edit_dis = 0.0
-        for (pred, pred_conf), (target, _) in zip(preds, labels):
+        edit_dis = 0
+        
+        for (pred, _), (target, _) in zip(preds, labels):
+            
             if self.ignore_space:
                 pred = pred.replace(" ", "")
                 target = target.replace(" ", "")
             if self.is_filter:
                 pred = self._normalize_text(pred)
                 target = self._normalize_text(target)
+
             norm_edit_dis += Levenshtein.normalized_distance(pred, target)
+            edit_dis += Levenshtein.distance(pred, target)
```

Comment:
As it is shown in the doc normalize_distance = distance / max, therefore they are  interchangeable. So why do we need to add distance on top of the normalize_distance we already have?

refer: https://maxbachmann.github.io/RapidFuzz/Usage/distance/Levenshtein.html#rapidfuzz.distance.Levenshtein.normalized_distance

Reply:
Hey @shiyutang, first of all here is [link](https://towardsdatascience.com/evaluating-ocr-output-quality-with-character-error-rate-cer-and-word-error-rate-wer-853175297510#5aec) to an article about the metric I'd like to implement.

As explained there **CER** calculation is based on the concept of **Levenshtein** distance, and is defined as :

CER<sub>normalized</sub> = S+D+I / S+D+I+C

* S + D + I, being **Levenshtein** distance (not normalized) referred as `edit_dis`in the code
* C, being number of correct characters, referred as `correct_num` in the code

where:
   * S = Number of Substitutions
   * D = Number of Deletions
   * I = Number of Insertions

Simplifying a bit would give us either :
* CER<sub>normalized</sub> = `Levenshtein.distance(pred, target)` / `Levenshtein.distance(pred, target)` + C
or
* CER<sub>normalized</sub> = `Levenshtein.distance(pred, target)` / `len(target)` + I

I totally agree that computing Levenshtein distance twice isn't a great solution, and given the way it is implemented in RapidFuzz, i can rework my PR to compute either non normalized Levenshtein distance **OR** normalized Levenshtein distance and use [`_levenshtein_maximum`](https://github.com/maxbachmann/RapidFuzz/blob/main/src/rapidfuzz/distance/Levenshtein_py.py#L14) method to compute the `max_distance`.